genesis/src/lib.rs

```diff
@@ -198,7 +203,7 @@ impl ValidatorPath {
 #[repr(transparent)]
 pub struct GenesisTransactionBuilder {
     /// Instructions
-    isi: Vec<InstructionExpr>,
+    pub isi: Vec<InstructionExpr>,
```

Comment:
make it private again after applying previous comment

Reply:
just reverting this since we do not believe in modifying genesis.